ClubManagement/ClubManagement/Models/UserLoginEventModel.cs

```diff
@@ -11,16 +11,16 @@ public UserLoginEventModel()
 
         public UserLoginEventModel(EventModel eventModel)
         {
-            base.Id = eventModel.Id;
-            base.Title = eventModel.Title;
-            base.Description = eventModel.Description;
-            base.Time = eventModel.Time;
-            base.CreatedBy = eventModel.CreatedBy;
-            base.CreatedTime = eventModel.CreatedTime;
-            base.Latitude = eventModel.Latitude;
-            base.Longitude = eventModel.Longitude;
-            base.Place = eventModel.Place;
-            base.ImageUrl = eventModel.ImageUrl;
+            Id = eventModel.Id;
+            Title = eventModel.Title;
+            Description = eventModel.Description;
+            TimeStart = eventModel.TimeStart;
```

Comment:
Please add `TimeEnd` to here and other places

Reply:
Done